llvm/test/tools/llvm-objdump/X86/elf-pgoanalysismap.yaml

```diff
@@ -48,6 +48,8 @@ Symbols:
 # RUN: yaml2obj %s --docnum=2 -o %t2
 # RUN: llvm-objdump %t2 -d --symbolize-operands --no-show-raw-insn --no-leading-addr | \
 # RUN:   FileCheck %s --check-prefix=ENTRYCOUNT-BLOCKFREQ
+# RUN: llvm-objdump %t2 -d --symbolize-operands --pretty-pgo-analysis-map --no-show-raw-insn --no-leading-addr | \
+# RUN:   FileCheck %s --check-prefix=ENTRYCOUNT-BLOCKFREQ-PRETTY
```

Comment:
I think when checking the formatting of a new dumping option, you should probably use `--strict-whitespace` and `--match-full-lines`. In some cases, it even makes sense to do `--implicit-check-not={{.}}`.

Reply:
Added `--strict-whilespace` and `--match-full-lines`.